File: Aug 28/D_Remove_Two_Letters.py

def solve():
    n = int(input())
    s = input().strip()
    st = set()
    cnt = 0
    for i in range(0,n-2):
        if s[i] == s[i+2]:
            cnt += 1
    
    print(n - cnt - 1)

t = int(input())
while t:
    solve()
    t-=1

File: Aug 28/D_Remove_Two_Letters.cpp
#include<bits/stdc++.h>
using namespace std;

void solve(){
    int n;
    cin >> n;
    string s;
    cin >> s;
    int cnt = 0;
    for(int i=0; i<n-2; i++){
        if(s[i] == s[i+2]) cnt++;
    }
    cout<<n - cnt - 1 << endl;
}

int main(){
    int t;
    cin >> t;
    while(t--) solve();
    return 0;
}